notebooks/day1/03-exercises.jl

```diff
@@ -356,6 +353,9 @@ begin
 
 		""",
 		questions = @safe[q31, q32, q33],
+		hints = [
```

Comment:
added a hint here, because I found it to be easier to start the questions that way